lib/src/commit.rs

```diff
@@ -18,7 +18,8 @@ use std::hash::{Hash, Hasher};
 use std::sync::Arc;
```

Comment:
Re commit message:

>  We don't yet a way for the user
to say they want to use tree-level conflicts even for these
cases. However, since the backend can decide, we should be able to
have our backend return tree-level conflicts. All writes will still
use path-level conflicts, so the experimentation we can do at Google
is limited.

Is the plan to introduce a config option, or is it too early for that?